crates/loam-cli/src/commands/dev/mod.rs

```diff
@@ -0,0 +1,105 @@
+use clap::Parser;
+use notify::{RecursiveMode, Watcher};
+use std::path::Path;
+use tokio::time;
+
+use crate::commands::build;
+
+use super::build::clients::LoamEnv;
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub build_cmd: build::Cmd,
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Watcher(#[from] notify::Error),
+    #[error(transparent)]
+    Build(#[from] build::Error),
+    #[error("IO error: {0}")]
+    Io(#[from] std::io::Error),
+}
+
+impl Cmd {
+    pub async fn run(&self) -> Result<(), Error> {
+        // Set LOAM_ENV to development
+        std::env::set_var("LOAM_ENV", "development");
```

Comment:
I think we should _default_ `LOAM_ENV` to `development` _when you use `loam dev`_. And when you use `loam build`, we should default it to `production`. But you should be able to override the defaults.

I think this is the behavior of other commands like `astro dev` and `astro build`.

If you want to run the watcher but compile everything for `LOAM_ENV=production`, why should we stop you?

Reply:
Good catch, I should have deleted this line. It is currently defaulting to development later when it runs build.